lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "2.9.0-SNAPSHOT",
+  "version": "2.8.2-SNAPSHOT",
```

Comment:
Should we keep the version as "2.9.0-SNAPSHOT" in main branch?

Sorry I think I've asked this question before, and forget if we decided it doesn't matter?

Reply:
yes we should, thanks for always seeing and asking. I didn't realize since no conflict shown, let me fix real quick

Reply:
Thanks 🙂 Would it be helpful if we extracted the part of our deploy workflow that bumps the version into a separate script so you could run it when creating these PRs?

Reply:
I like it with the deploy because it gets it right on the branch, I just need to remember when moving stuff between branches.  Thanks again for catching it because it made me look at the package.jsons too which had maintenance branch version.

Reply:
it may help if callable separately as well as by deploy workflow